recipe/meta.yaml

```diff
@@ -19,24 +19,26 @@ build:
     - omicron-process = omicron.cli.process:main
     - omicron-root-merge = omicron.cli.root_merge:main
     - omicron-status = omicron.cli.status:main
+    - omicron-archive = omicron.cli.archive:main
+    - omicron-merge-with-gaps = omicron.cli.merge_with_gaps:main
 
 requirements:
   host:
     - pip
-    - python >=3.6
+    - python >=3.9
     - setuptools
   run:
     - dqsegdb2
     - gwdatafind
-    - gwpy >=1.0.0
+    - gwpy >=2.0.0
     - h5py
     - ligo-segments
     - lscsoft-glue >=1.60.0
     - markuppy
     - numpy
-    - omicron >=2.3.12
+    - omicron
```

Comment:
Why was this version pin removed? This used to be a requirement in pyomicron (which couldn't be specified in the python metadata), was backwards-compatibility extended?

Reply:
This was a mistake. Minor change were made for 3.0

Reply:
That is done